source/_integrations/sql.markdown

```diff
@@ -220,6 +220,24 @@ LIMIT
 Replace `-1 day` with the target offset, for example, `-1 hour`.
 Use `state` as column for value.
 
+#### for MariaDB
+Since the ```strftime``` function doesn't exist in MariaDB you need to use the following query
+```sql
+SELECT
+  states.state
+FROM
+  states
+  INNER JOIN states_meta ON
+    states.metadata_id = states_meta.metadata_id
+WHERE
+  states_meta.entity_id = 'sensor.temperature_in'
+  AND last_updated_ts <= (UNIX_TIMESTAMP() - 24 * 3600)
+ORDER BY
+  last_updated_ts DESC
+LIMIT
+  1;
+```
+
```

Comment:
Rather than duplicate the whole statement, I'd only document the where clause. Some slight improvement to make the SQL more readable:

```suggestion
#### MariaDB

On MariaDB the following where clause can be used to compare the timestamp:

```sql
...
  AND last_updated_ts <= UNIX_TIMESTAMP(NOW() - INTERVAL 1 DAY)
...

Replace `- INTERVAL 1 DAY` with the target offset, for example, `- INTERVAL 1 HOUR`.
```